grant-project/grant-monorepo
# Redux dev tools in production

Pretty self-explanatory.

### Steps to Test

* Run `yarn build`
* Run `yarn start`
* Confirm redux dev tools still works
